text/metrics/0128-instrument-multi-aggregators.md

```diff
@@ -0,0 +1,88 @@
+# SDK Proposal: Default and Configurable Aggregations for Instruments
+
+This proposal builds off the work of John in
+[OTEP-126](https://github.com/open-telemetry/oteps/pull/126) and Chris in
+[OTEP-89](https://github.com/open-telemetry/oteps/pull/89).
+
+The unique part of this proposal is that all metric exports are based on
+views. Each `Instrument` has a default `View` created when the `Instrument` is
+created.
+
+## Definitions
+
+* Measurement: The value recorded along with a set of label keys and values.
+* Instrument: The thing a `Measurement` is associated with when recorded. Each
+  has a name, description, instrument kind and unit.
+* Aggregation: Defines how a set of measurements for an instrument are to be
+  combined for export.
+* View: An aggregation on an `Instrument` and some metadata (a name,
+  description, etc).
+
+## View
+
+* name: The name used when exporting metrics. 
+* description: String value describing what the aggregated measurements represent.
+* aggregation: The implementation of combining measurements to use.
+* labels: The labels to filter down to from the labels included in the measurement.
+* exporter: Each `view` can be associated with an exporter. By default it is set
```

Comment:
How will specifying the exporter as part of the view work with in the whole  record -> collect -> export pipeline? Is specifying the exporter through views the recommended way of exporting now (instead of a PushController -> start_pipeline mechanism). If this is the case, will I need to define a View with an exporter even though I just want to export a simple metric (with no View functionality)?

Reply:
@lzchen this proposal is that any instrument is only aggregated and thus exportable through a view, but there is a default view created automatically for every created instrument, so it results in the same outcome as we have today without it but is now through a view.

Reply:
@tsloughter 
What is the default SDK exporter? Also, if a user wants to use something other than the default exporter, they have to create a View?

Reply:
A view is always used, even if only using a single exporter.

The "default exporter" in the case of only a single exporter would be just that exporter. I don't know what the spec would actually look like for supporting multiple exporters, this is currently a "todo" it seems in the metrics SDK PR. I figure it'd be kind of language specific. In Erlang we'd probably just either have users give a single exporter or a map of exporters where use of the atom `default` is given for the exporter to use when none is given to the view definition.

Reply:
So the default exporter would be something that users set correct? And then Views (if exporter is not specified), would use that/pick that up somehow?

Reply:
Correct.